src/js/ScoreForm.js

```diff
@@ -1,23 +1,23 @@
 export default class ScoreForm {
   constructor() {
-    this.name = '';
+    this.user = '';
     this.score = '';
   }
 
-  setEventListener(leaderboard) {
+  setEventListener(api) {
     const submitButton = document.querySelector('form button');
-    submitButton.addEventListener('click', (e) => this.submitScore(e, leaderboard));
+    submitButton.addEventListener('click', (e) => this.submitScore(e, api));
   }
 
-  submitScore(e, leaderboard) {
+  submitScore(e, api) {
     e.preventDefault();
     this.createScore();
-    leaderboard.add(this);
+    api.post(this);
   }
```

Comment:
[OPTIONAL]

- Name and score fields should be properly validated to ensure user cannot submit blank entries. :smile: